src/tribol/search/InterfacePairFinder.cpp

```diff
@@ -53,7 +53,8 @@ bool geomFilter( InterfacePair const & iPair, ContactMode const mode )
    MeshData& mesh1 = meshManager.GetMeshInstance(meshId1);
    MeshData& mesh2 = meshManager.GetMeshInstance(meshId2);
 
-   int dim = ( mesh1.m_elementType == tribol::EDGE ) ? 2 : 3;
+//   int dim = ( mesh1.m_elementType == tribol::EDGE ) ? 2 : 3;
```

Comment:
Why is this line commented out?  Does it still need to be here?

Reply:
Removed. Doesn't need to be there.